### ISSUE TITLE
Where can I find "Test set"?

### ISSUE THREAD

**Issue by myteen2** [oleschum/radar_scenes#13]
First of all, thank you for sharing wonderful data set. 
I want to test and compare DL algorithms based on RadarScenes but I can only find "train" and "validation" in the category. 
How did you compute the results reported in your papers?
Where can I find the test set?

**Comment by oleschum** [comment]
You are welcome!

We did not split the data set into three parts (training, validation, test), but only made a proposal for training and validation. You can use the "training" files for your complete training procedure (including hyper param tuning, if needed). The validation set is then the one you should use to report the scores of your algorithm (often called test set).

**Comment by myteen2** [comment]
Thank you for your answer, and I'm sorry for the late response.

To confirm, I will rewrite what I understand below.

When I evaluate the final performance of a model using sequences marked as "validation" in data/sequences.json, 
that's the test-set result in the three-division perspective (training, validation, test). 
This is the value that allows fair comparison with the previous papers that have reported performance on RadarScenes.

If there is anything I misunderstood, please correct it.
Once again, thank you for the answer.
